src/common/SearchEmptyState.js

```diff
@@ -0,0 +1,55 @@
+/* @flow */
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import { ZulipButton, Label } from './';
+
+const styles = StyleSheet.create({
```

Comment:
Yeah, it is mostly to avoid conflicts. We might want to do componentStyles everywhere, but we haven't decided on it, so it is OK as is.